Sources/Public/Animation/AnimationPublic.swift

```diff
@@ -69,7 +69,7 @@ extension Animation {
       return animation
     } catch {
       /// Decoding error.
-      print(error)
+      LottieLogger.shared.assertionFailure("Error when decoding animation \"\(name)\": \(error)")
```

Comment:
Wondering if you think this should be a `warn`? I could imagine a case where you want to recover from an invalid animation and not have it crash in debug e.g. if you're writing a lottie viewer app

Reply:
Sure

Reply:
```suggestion
      LottieLogger.shared.warn("Error when decoding animation \"\(name)\": \(error)")
```